{"name": "Тест на проверку функции калькулятора - деление с числами внутри диапазона integer", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидается -7, получено -1\nassert -1 == -7\n  --1\n  +-7", "trace": "setup = None, param_test = ({'left_operand': 2013914191, 'operation': '/', 'right_operand': -272083899}, -7.401813184836785)\n\n    @allure.title(\"Тест на проверку функции калькулятора - деление с числами внутри диапазона integer\")\n    def test_division_for_integers_within_borders(setup, param_test):\n        (input_data, expected_data) = param_test\n        with allure.step(f\"отправляем запрос с входными данными {input_data} и получаем результат расчёта\"):\n            result = do_request_and_get_response(input_data)\n        expected_result = int(expected_data)\n        with allure.step(f\"Проверяем, что полученный результат {result} = ожидаемому {expected_result}\"):\n>           assert result == expected_result, f\"Ожидается {expected_result}, получено {result}\"\nE           AssertionError: Ожидается -7, получено -1\nE           assert -1 == -7\nE             --1\nE             +-7\n\nexpected_data = -7.401813184836785\nexpected_result = -7\ninput_data = {'left_operand': 2013914191, 'operation': '/', 'right_operand': -272083899}\nparam_test = ({'left_operand': 2013914191, 'operation': '/', 'right_operand': -272083899}, -7.401813184836785)\nresult     = -1\nsetup      = None\n\ntests\\test_division.py:12: AssertionError"}, "steps": [{"name": "отправляем запрос с входными данными {'left_operand': 2013914191, 'operation': '/', 'right_operand': -272083899} и получаем результат расчёта", "status": "passed", "start": 1579356201322, "stop": 1579356201509}, {"name": "Проверяем, что полученный результат -1 = ожидаемому -7", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидается -7, получено -1\nassert -1 == -7\n  --1\n  +-7\n", "trace": "  File \"C:\\Users\\Leonid\\PycharmProjects\\ModulBank\\tests\\test_division.py\", line 12, in test_division_for_integers_within_borders\n    assert result == expected_result, f\"Ожидается {expected_result}, получено {result}\"\n"}, "start": 1579356201509, "stop": 1579356201510}], "parameters": [{"name": "param_test", "value": "({'left_operand': 2013914191, 'operation': '/', 'right_operand': -272083899}, -7.401813184836785)"}], "start": 1579356201321, "stop": 1579356201511, "uuid": "3fa80da4-b573-45c0-aa91-b01a0c05bd2c", "historyId": "92ecb419375620850a0fa5f9bde0ae9d", "testCaseId": "9f6ca8f533d02c3e239a5a6be76bb0f5", "fullName": "tests.test_division#test_division_for_integers_within_borders", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_division"}, {"name": "host", "value": "DESKTOP-1R5TURF"}, {"name": "thread", "value": "3576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_division"}]}